doc/source/notebooks/basics/classification.ipynb

```diff
@@ -344,9 +346,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In this table, the first two lines are associated to the kernel parametres, and the last two correspond to the variational parameters. Note that in practice, $q_\\Sigma$ is actually parametrised by its lower-triangular square root $q_\\Sigma = q_{sqrt} q_{sqrt}^T$ in oder to ensure it's positive definiteness.\n",
+    "In this table, the first two lines are associated to the kernel parameters, and the last two correspond to the variational parameters. Note that in practice, $q_\\Sigma$ is actually parametrised by its lower-triangular square root $q_\\Sigma = q_\\text{sqrt} q_\\text{sqrt}^T$ in oder to ensure its positive-definiteness.\n",
     "\n",
-    "For more details on how to handle models in GPflow (getting/setting parameters, fixing some of them during optim, using priors...), we refer the reader to the [notebook on models](../understanding/models.ipynb)."
+    "For more details on how to handle models in GPflow (getting/setting parameters, fixing some of them during optimisation, using priors...), we refer the reader to the [notebook on models](../understanding/models.ipynb)."
```

Comment:
```suggestion
    "For more details on how to handle models in GPflow (getting/setting parameters, fixing some of them during optimisation, using priors...), we refer the reader to the [notebook on GPflow models](../understanding/models.ipynb)."
```

Reply:
The problem is that the notebook doesn't exist (doesn't have contents)...

Reply:
Maybe link to this page in the meantime?

https://gpflow.readthedocs.io/en/develop/intro.html#what-models-are-implemented